README.md

```diff
@@ -140,4 +155,5 @@ We're using Google's auto-formatter to format the code. The build pipeline is se
 
 This project adheres to the [Open Code of Conduct][code-of-conduct]. By participating, you are expected to honor this code.
 
+[mkdocs]: https://spotify.github.io/mobius/
```

Comment:
This will be the website url if ya'll decide to migrate over to GitHub Pages 🤞

Of course, you can override it with your custom domain if you wish.